.github/workflows/tests.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       uses: shivammathur/setup-php@verbose
       with:
         php-version: ${{ matrix.php }}
-        extensions: grpc
+        extensions: grpc-1.49.0
```

Comment:
Wasn't this breaking on the main branch too?
If so, then this really needs to be tackle in a PR from `main`.

Reply:
This was breaking in main, I got this idea from our main branch only.

Reply:
This is the PR on main: https://github.com/googleapis/google-cloud-php/pull/5696/files

Reply:
Well, if it's done, we don't really need to do it again. We just need to sync main into our branch(gcs-conformance-testing).

Reply:
Done